frontend/src/bundles/auth/components/auth-layout/styles.module.scss

```diff
@@ -1,22 +1,19 @@
 .container {
     flex-direction: column;
     width: 100%;
-    min-height: 100vh;
+    min-height: calc(100vh - 60px);
```

Comment:
```suggestion
    min-height: 100vh;
```
I think this page should be used without the sidebar and header as in the design.
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/42421305/3258f4e4-67ca-4cfe-8d0a-28e8c748a78f)

Reply:
Good catch. But I think this should be a different task? Since it will touch on routing. 
Will have to ask about what best to do on this one.

Reply:
I wrote that it's not about routing. I wrote that this page should be 100vh, not calculating something.

Reply:
This will be addressed when this PR gets merged:
https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/issues/276

-60px will be removed